README.md

```diff
@@ -12,11 +12,14 @@ Requires:
 ### Configuration
 Configuration values can be set either from environmental variables or code:
 
-`SIGNALFX_SERVICE_NAME` / [WithServiceName](https://godoc.org/github.com/signalfx/signalfx-go-tracing/tracing/#WithServiceName) Name identifying the service as a whole (defaults to `SignalFx-Tracing`)
-
-`SIGNALFX_ENDPOINT_URL` / [WithEndpointURL](https://godoc.org/github.com/signalfx/signalfx-go-tracing/tracing/#WithEndpointURL) URL to send traces to (defaults to `http://localhost:9080/v1/trace`)
-
-`SIGNALFX_ACCESS_TOKEN` / [WithAccessToken](https://godoc.org/github.com/signalfx/signalfx-go-tracing/tracing/#WithAccessToken) (no default)
+| Code | Environment Variable | Default Value | Notes |
+| ---  | ---                  | ---           | ---   |
+| [WithServiceName](https://godoc.org/github.com/signalfx/signalfx-go-tracing/tracing/#WithServiceName) | `SIGNALFX_SERVICE_NAME` | `SignalFx-Tracing` | Name identifying the service as a whole |
+| [WithEndpointURL](https://godoc.org/github.com/signalfx/signalfx-go-tracing/tracing/#WithEndpointURL) | `SIGNALFX_ENDPOINT_URL` | `http://localhost:9080/v1/trace` | URL to send traces to  |
```

Comment:
To confirm, can this be a Smart Agent, SignalFx endpoint, or OpenTelemetry Collector? If so, let's specify here.

Suggestion for existing text:

`The URL to send traces to.`

Reply:
Yes, it can send it to all of those backends.